v4-client-py-v2/dydx_v4_client/indexer/rest/modules/account.py

```diff
@@ -335,3 +425,199 @@ async def get_historical_trading_rewards_aggregated(
+        ticker: Optional[str] = None,
+        side: Optional[OrderSide] = None,
+        status: Optional[OrderStatus] = None,
+        order_type: Optional[OrderType] = None,
+        good_til_block_before_or_at: Optional[int] = None,
+        good_til_block_time_before_or_at: Optional[str] = None,
+        return_latest_orders: Optional[bool] = None,
+    ):
+        """
+        Query for orders filtered by order params of a parent subaccount.
+
+        Args:
+            address (str): The account address.
+            subaccount_number (int): The subaccount number.
+            limit (Optional[int]): The maximum number of orders to retrieve.
+            ticker (Optional[str]): The ticker filter.
+            side (Optional[OrderSide]): The order side filter.
+            status (Optional[OrderStatus]): The order status filter.
+            order_type (Optional[OrderType]): The order type filter.
+            good_til_block_before_or_at (Optional[int]): The block number filter for orders good until before or at.
+            good_til_block_time_before_or_at (Optional[str]): The timestamp filter for orders good until before or at.
+            return_latest_orders (Optional[bool]): Whether to return only the latest orders.
+
+        Returns:
+            Any: The aggregated list of parent orders.
+        """
+        uri = f"/v4/orders/parentSubaccountNumber"
+        return await self.get(
+            uri,
+            params={
+                "address": address,
+                "parentSubaccountNumber": subaccount_number,
+                "limit": limit,
+                "ticker": ticker,
+                "side": side,
+                "status": status,
+                "type": order_type,
+                "goodTilBlockBeforeOrAt": good_til_block_before_or_at,
+                "goodTilBlockTimeBeforeOrAt": good_til_block_time_before_or_at,
+                "returnLatestOrders": return_latest_orders,
+            },
+        )
+
+    async def get_parent_fills(
+        self,
+        address: str,
+        subaccount_number: int,
+        limit: Optional[int] = None,
+        created_before_or_at_height: Optional[int] = None,
+        created_before_or_at: Optional[str] = None,
+        market: Optional[str] = None,
+        market_type: Optional[TickerType] = None,
+    ):
+        """
+        Query for fills (i.e. filled orders data).
+
+        Args:
+            address (str): The account address.
+            subaccount_number (int): The subaccount number
+            limit (Optional[int]): The maximum number of fills to retrieve.
+            ticker (Optional[str]): The ticker filter.
+            market_type (TickerType): The market type filter.
+            created_before_or_at_height (Optional[int]): The block height filter for fills created before or at.
+            created_before_or_at (Optional[str]): The timestamp filter for fills created before or at.
+
+        Returns:
+            Any: The fills data.
+        """
```

Comment:
The documentation is differs with function arguments, consider making it consistent